docs/walkthrough.md

```diff
@@ -111,38 +111,38 @@ check out the "Cleaning Up" section below. Follow those instructions before you
 
 The service catalog API has five main concepts:
 
-- Broker Server: A server that acts as a service broker and conforms to the 
+- ServiceBroker Server: A server that acts as a service broker and conforms to the 
 [Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md)
 specification. This software could be hosted within your own Kubernetes cluster
 or elsewhere.
 
 The remaining four concepts all map directly to new Kubernetes resource types
 that are provided by the service catalog API.
 
-- `Broker`: An in-cluster representation of a broker server. A resource of this
+- `ServiceBroker`: An in-cluster representation of a broker server. A resource of this
 type encapsulates connection details for that broker server. These are created
 and managed by cluster operators who wish to use that broker server to make new
 types of managed services available within their cluster.
 - `ServiceClass`: A *type* of managed service offered by a particular broker.
-Each time a new `Broker` resource is added to the cluster, the service catalog
+Each time a new `ServiceBroker` resource is added to the cluster, the service catalog
 controller connects to the corresponding broker server to obtain a list of
 service offerings. A new `ServiceClass` resource will automatically be created
 for each.
-- `Instance`: A provisioned instance of a `ServiceClass`. These are created
+- `ServiceInstance`: A provisioned instance of a `ServiceClass`. These are created
 by cluster users who wish to make a new concrete _instance_ of some _type_ of
 managed service to make that available for use by one or more in-cluster
-applications. When a new `Instance` resource is created, the service catalog
+applications. When a new `ServiceInstance` resource is created, the service catalog
 controller will connect to the appropriate broker server and instruct it to
 provision the service instance.
-- `Binding`: A "binding" to an `Instance`. These are created by cluster users
-who wish for their applications to make use of a service `Instance`. Upon
+- `ServiceInstanceCredential`: A "binding" to an `ServiceInstance`. These are created by cluster users
+who wish for their applications to make use of a service `ServiceInstance`. Upon
 creation, the service catalog controller will create a Kubernetes `Secret`
 containing connection details and credentials for the service instance. Such
 `Secret`s can be mounted into pods as usual.
 
 These concepts and resources are the building blocks of the service catalog.
 
-## Step 3 - Installing the UPS Broker
+## Step 3 - Installing the UPS ServiceBroker
```

Comment:
As a meta comment, we should later decide what exactly we want our messaging should be. I think there's a distinction between a "service broker" (the overall concept) and "ServiceBroker" (the API server resource). On top of that, I also think it's okay to have discussions about "brokers" (minus the "service" aspect) if the context is clear.

Definitely not in the scope of this PR, but I think we should later go through our docs with a fine-toothed comb with those sorts of things in mind.